### ISSUE TITLE
Help Billy's team clean up School Zone Beacon data

### ISSUE THREAD

**Issue by amenity** [cityofaustin/atd-data-tech#3472]
The School Zone Beacon data is out of date and doesn't contain precise coordinates for each beacon. Billy will assign resources to cleaning the data, but we need to provide them with a tool + process for doing so. 
- Updating coordinates
- Installation date, if possible

**Comment by amenity** [comment]
@johnclary - coordinate with @jaime-mckeown to set up a data collection/update process.

**Comment by jaime-mckeown** [comment]
- The coordinates can be updated in Knack after the points have all been moved to the correct locations.
- @amenity @johnclary should I add an installation date to the GIS audit data?

**Comment by amenity** [comment]
@jaime-mckeown — Thanks!

I would say yes to the installation date —  thinking that getting in the field along with the coordinates is our best bet for populating that data (somewhat) accurately. I think they will be guesstimating a lot of them, but probably better guesstimates if they're there in person? cc/ @johnclary.

**Comment by jaime-mckeown** [comment]
Installation Date has been added to the SZB Feature Layer.

**Comment by amenity** [comment]
Thanks, @jaime-mckeown . @johnclary — next steps here?

**Comment by jaime-mckeown** [comment]
@amenity I'm going to meet with Billy this week to make sure the Web App is capturing everything they need, then schedule training with the 2 tech's that will be doing the work.

**Comment by amenity** [comment]
@jaime-mckeown — gotcha, sounds great. LMK if you need anything else from me!

**Comment by jaime-mckeown** [comment]
Went over data and Web Application with Billy. He will be letting me know who to invite to training, hoping to schedule for next week.

**Comment by johnclary** [comment]
yayyy! thanks @jaime-mckeown! let me know if you feel like you need a hand with training. happy to sit in.

**Comment by jaime-mckeown** [comment]
@johnclary I have a meeting with Jacquie and Cathy from the Addressing Team next week to discuss the possibility of creating official addresses for the Beacons. I'll forward you the email chain, if I missed anything please let me know so I can communicate it to them next week. Thanks!

**Comment by jaime-mckeown** [comment]
@amenity @johnclary I just talked to Billy about setting up training to get the Signal Tech's going on verifying the data. He is curious what the expected time frame is for this audit. This will affect how many Signal Tech's he involves with the process, therefore who needs training. Thanks!

**Comment by jaime-mckeown** [comment]
Billy, Edward, and myself to meet next Monday to discuss how best to add fields in order to denote which beacon each Web-IO is located on and what beacon clocks it controls. My suggestion was to add 2 fields, one to show if the Web-IO is located on that beacon (Yes/No), and the other to show which beacon clocks are controlled by what Web-IO's (IP Address, serial #, etc.). Billy agreed with my logic, but wants to include Edward as he is the expert on the Web-IO's.

**Comment by jaime-mckeown** [comment]
Met with Billy, Edward, Nick, and Larry to discuss the school zone beacons.
- Edward informed us that there are Clocks that are not located on beacons, they can be installed in a box that is mid-way between 2 beacons.
- Web-IOs are located in cabinets at signalized intersections. They can control up to 60 clocks. There are a few Web-IO's located on the top of the OTC building.
- Will be discussing with John tomorrow for how best to configure this to work between GIS and Data Tracker.

**Comment by jaime-mckeown** [comment]
**Notes from Discussion with John**
- Create a separate feature class for Web-IO's, need the following fields: Latitude, Longitude, Description, IP Address, Communication?, Signal Cabinet ID?, plus any other fields deemed necessary by Signals.
- Add a new field to School Zone Beacons feature class that includes the following choices: Beacon, Beacon-Clock, Clock. Need to figure out best field name.
- John mentioned the Locations Data Tracker dataset, we may be able to use this layer for locations, noted for the future.

- Sent Edward email to request his spreadsheet of Web-IO's so I can create the feature class.

**Comment by jaime-mckeown** [comment]
- Received Wed-IO List from Edward
- Billy responded to the email, agreed with what John and I talked about above.
- Created new Feature Class called SZB_Web_IO, all have been added except the one's I have questions waiting on from Edward.
- Created new Domain for Asset Type, created new field called ASSET_TYPE and coded to Beacon if CLOCK = No and Beacon-Clock if CLOCK = Yes. There is also an option for just Clock if it is not on a Beacon.

**Comment by jaime-mckeown** [comment]
Could never get my questions answered by Edward regarding Web-IOs. Billy reached out to Brian C. and he was able to help me map the last few that had questions. Can now finally publish the Web-IO layer to AGOL.

**Comment by jaime-mckeown** [comment]
Met with Billy to go over the latest changes. He requested that fields are added in both layers to connect the Web-IOs to the Clocks and vis-versa.

**Comment by jaime-mckeown** [comment]
Email to Brian and Billy last thursday:

> Billy and I met earlier, and he was asking if I can show which Web-IO is connected to each Clock. We were looking at the Images you emailed me on Monday, and they showed the connections between the two. Would it be possible for you to send me an export from Zonewatch that shows every Web-IO and all Clocks that they feed so I can populate the data?
> 
> Billy, John said that since the GIS data will be fed into Data Tracker that we only need to add the Web-IOs to the beacon layer connecting them to each Clock. All of the relationships will be created between Web-IOs, Clocks, and Beacons in Data Tracker.
>

**Comment by johnclary** [comment]
@jaime-mckeown plz follow up

**Comment by jaime-mckeown** [comment]
Sent a follow-up email to Brian regarding Web-IO and Clock data connections from Zonewatch.

**Comment by jaime-mckeown** [comment]
Waiting for Web-IO / Clock data connections from Brian.

**Comment by jaime-mckeown** [comment]
Emailed Brian again requesting the Web-IO / Clock data connections.

**Comment by jaime-mckeown** [comment]
Brian sent me a Spreadsheet for connecting the Web-IO's to the SZB Clocks. It didn't cover all of them, but it will help the Signal Tech's get started. They will need to verify all that have been coded and find all other connections during the Audit Project. I will email Billy to let him know that the Web Map is ready for them to take over to start working on the SZB Audit.

**Comment by jaime-mckeown** [comment]
New Issue for Training, closing this Issue.
